src/RealityDataClient.ts

```diff
@@ -207,14 +207,17 @@ export class RealityDataAccessClient implements RealityDataAccess {
 
   private extractContinuationToken(url: string | undefined): string | undefined {
     if (url) {
-      const continuationToken = url.split("&continuationToken=");
-      return continuationToken[continuationToken.length - 1];
+      const pattern = /continuationtoken=[a-zA-Z0-9]*/i;
```

Comment:
ContinuationToken are opaque to the caller and we should not assume which char they may contain. Using URLSearchParams would be more robust. https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/get#examples